README.md

```diff
@@ -145,73 +145,64 @@ in the Minishift docker environment, as detailed in next section.
 dev-scripts/docker_create_images_and_tag_in_minishift.sh
 ```
 
-## How to deploy or roll-update it in Minishift
+## How to deploy or roll-update it in Minishift or OSIO
     
 ### Deployment prerequisites
 
-* Get [minishift](https://github.com/minishift/minishift#installation) (we have tested with v1.0.0beta4)
-  * Minishift v1.0.0beta4 includes the `oc` binary, and this should be added to your `$PATH`. The binary is located
-    in the `.minishift/cache/oc/<oc version>/` directory (e.g. on Linux, it is in `~/.minishift/cache/oc/v1.4.1/`).
-* Get [gofabric8](https://github.com/fabric8io/gofabric8#getting-started) (we have tested with v0.4.121)
-* Clone fabric8-online git repository:
+* [minishift](https://github.com/minishift/minishift#installation) (v1.0.0 or greater)
+* `oc` the OpenShift command line client tool. The binary is included in minishift distribution and can be found in folder `.minishift/cache/oc/<oc version>/`. You need to add that path to your `$PATH` environment variable.
```

Comment:
Might be worth noting that you can add `oc` to your path through minishift via `eval $(minishift oc-env)`.